src/sections/footer.html

```diff
@@ -1,4 +1,30 @@
-<div>
-    <h1>Footer</h1>
-    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam, quibusdam.</p>
-</div>
\ No newline at end of file
+<!-- Parent -->
+<div class="bg-webGrey px-5 lg:px-9 pt-20 pb-9">
+  <!-- Image   -->
+  <div class="flex justify-center w-full">
```

Comment:
remove unnecessary comments
```suggestion
<div class="bg-webGrey px-5 lg:px-9 pt-20 pb-9">
  <div class="flex justify-center w-full">
```